composer.json

```
@@ -3,6 +3,7 @@
     "description": "Storekeeper plugin",
     "require": {
         "php": "^7.4|^8.1|^8.2",
+        "ext-uuid": "*",
```

```suggestion
        "symfony/polyfill-uuid": "*",
```
better cos no os dependency